.ci/esy-check-hygiene.yml

```
@@ -3,6 +3,14 @@
 steps:
   - script: npm install -g esy@0.6.2
     displayName: 'npm install -g esy@0.6.2'
+  - script: npm install -g clang-format
+    displayName: 'npm install -g clang-format'
+  - script: clang-format -i src/wrapped/lib/*.c
+    displayName: 'clang-format -i src/wrapped/lib/*.c'
+  - script: clang-format -i src/wrapped/c/*.c
+    displayName: 'clang-format -i src/wrapped/c/*.c'
```

You forgot the header files:

```suggestion
  - script: clang-format -i src/wrapped/c/*.c
    displayName: 'clang-format -i src/wrapped/c/*.c'
  - script: clang-format -i src/wrapped/c/*.h
    displayName: 'clang-format -i src/wrapped/c/*.h'
```

---

Thanks for adding those!